code/game/objects/items/tools/crowbar.dm

```diff
@@ -42,6 +42,15 @@
 	icon_state = "crowbar"
 	toolspeed = 0.1
 
+/obj/item/crowbar/syndie
+	name = "wrench"
+	desc = "A small crowbar. This handy tool is useful for lots of things, such as prying floor tiles or opening unpowered doors."
+	icon_state = "crowbar"
+	toolspeed = 0.1
+
+/obj/item/crowbar/syndie/op
+	name = "syndicate wrench"
+	desc = "Specialized crowbar using superior materials that make it lightweight and sturdier."

```

Comment:
why bother making this if it's just a name/desc change lol

Reply:
I see I forgot to put in the icon state